ipalib/plugable.py

```diff
@@ -713,6 +713,15 @@ def finalize(self):
         self.__doing('finalize')
         self.__do_if_not_done('load_plugins')
 
+        if self.env.env_confdir is not None:
+            if self.env.env_confdir == self.env.confdir:
+                self.log.info(
+                    "IPA_CONFDIR env sets confdir to '%s'.", self.env.confdir)
+            else:
+                self.log.warn(
+                    "IPA_CONFDIR env is overridden by an explicit confdir "
+                    "argument.")
```

Comment:
This isn't what we agreed on on Monday ("raise runtime error" - although a `PublicError` subclass would be nicer). Please fix.